File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.cpp
class Solution {
public:
    double findMedianSortedArrays(vector<int>& nums1, vector<int>& nums2) {
        vector<int>nums3;
        for(int i=0;i<nums1.size();i++)
        {
            nums3.push_back(nums1[i]);
        }
        for(int i=0;i<nums2.size();i++)
        {
            nums3.push_back(nums2[i]);
        }
        int n=nums3.size();
        sort(nums3.begin(),nums3.end());
        if(n%2==0)
        {
            return ((nums3[n/2.0]+nums3[(n/2.0)-1])/2.0);
        }
        return (nums3[n/2.0]);
        
    }
};

File: 0004-median-of-two-sorted-arrays/0004-median-of-two-sorted-arrays.java
class Solution {
    public double findMedianSortedArrays(int[] nums1, int[] nums2) {
        List<Integer> list = new ArrayList<>();
        int i = 0;
        int j = 0;
        int n1 = nums1.length;
        int n2 = nums2.length;
        while(i < n1 && j < n2){
            if(nums1[i] < nums2[j]){
                list.add(nums1[i]);
                i++;
            }
            else{
                list.add(nums2[j]);
                j++;
            }
        }
        while(i < n1){
            list.add(nums1[i]);
            i++;
        }
        while(j < n2){
            list.add(nums2[j]);
            j++;
        }
        int n = n1 + n2;
        if(n % 2 == 1) return (float) list.get(n / 2);
        return (float) (list.get(n / 2) + list.get(n/2 - 1)) / 2;
    }
}